src/MusicalScore/VoiceData/Lyrics/LyricsEntry.ts

```diff
@@ -1,8 +1,9 @@
 import {LyricWord} from "./LyricsWord";
 import {VoiceEntry} from "../VoiceEntry";
+import { FontStyles } from "../../../Common";
```

Comment:
This should be from Common/FontStyles, since it can be ambiguous and has led to compile issues in the past. But I can also fix that in a follow-up commit.